src/NuGet.Services.AzureSearch/SearchDocumentBuilder.cs

```diff
@@ -8,6 +8,74 @@ namespace NuGet.Services.AzureSearch
 {
     public class SearchDocumentBuilder : ISearchDocumentBuilder
     {
+        public SearchDocument.LatestFlags LatestFlagsOrNull(VersionLists versionLists, SearchFilters searchFilters)
+        {
+            var latest = versionLists.GetLatestVersionInfoOrNull(searchFilters);
+            if (latest == null)
```

Comment:
I seem to have forgotten - in what cases would `latest` be null again? Is it if you have an incomplete `versionLists`? When does that happen? I ask as on line 65 you call `GetLatestVersionInfoOrNull` again, but that time you don't do a null check